client/src/components/RequestModal.js

```diff
@@ -231,9 +231,7 @@ export default class RequestModal extends Component {
                         onClick={this.submitRequest}
                         disabled={
                                 ((this.state.topicValue === '' || 
-                                this.state.availabilityValue === '') &&
-                                (this.state.topicValue === '' ||
-                                this.state.note === ''))
+                                this.state.availabilityValue === ''))
```

Comment:
The backend for requests is built upon having a time pretty aggressively - 
We can fix this by making any time related fields non mandatory, but spills over into email helpers and out of the scope of this PR. Until then, requests without a time will fail to either update or be added successfully

Reply:
Let's discuss potentially switching back to blocking time-absent requests again next Sunday